### PR TITLE
Add Ruby Together CTA

### PR DESCRIPTION
**Pull request by mrb**
/c @indirect

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ The Ruby community's gem host.
 * Create more transparent and accessible project pages
 * Enable the community to improve and enhance the site
 
+## Supporting
+
+<a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a><br/>
+  Bundler is maintained by <a href="https://rubytogether.org/">Ruby Together</a>, a grassroots initiative committed to supporting the critical Ruby infrastructure you rely on. Contribute today <a href="https://rubytogether.org/developers">as an individual</a> or even better, <a href="https://rubytogether.org/companies">as a company</a>, and ensure that Bundler, RubyGems, and other shared tooling is around for years to come.
```
Comment (dwradcliffe) [inline README.md]: Shouldn't this say RubyGems.org?

Comment (mrb) [inline README.md]: @dwradcliffe Sure! I'll add that.

Comment (dwradcliffe) [inline README.md]: specifically the first part "Bundler is maintained by"

Comment (mrb) [inline README.md]: @dwradcliffe D'oh! Yes. Will fix :)

Comment (mrb) [inline README.md]: @dwradcliffe Done!

**Pull request merged**